src/styles/style.css

```
@@ -0,0 +1,115 @@
+@import '../styles/stars.css';
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  --white: #FFF;
+  --dark-blue: #060B35;
+  --gray: #616070;
+  --blue: #00ACDC;
+}
+
+.card {
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid var(--white);
+  background: var(--white);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__title {
+  color: var(--dark-blue);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px 4px;
```

since we reset default margin for `h2` and `p` plus we add padding to our card let's use only margin-bottom 
```suggestion
  margin:-bottom: 4px;
```